.github/workflows/ci.yml

```
@@ -53,18 +58,50 @@ jobs:
+            | tee tarpaulin_output.txt
+
+      # 標準出力から「coverage,」を含む行を抜き出しファイル保存
+      - name: Extract coverage summary
+        run: |
+          grep 'coverage,' tarpaulin_output.txt > coverage_summary.txt \
+            || echo "No coverage info found" > coverage_summary.txt
+
+      # Pull Request 上のみコメントを投稿
+      - name: Post coverage comment
+        if: ${{ github.event_name == 'pull_request' }}
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)